src/components/activity/ActivityTableMenu.svelte

```diff
@@ -1,10 +1,11 @@
 <svelte:options immutable={true} />
 
 <script lang="ts">
+  type T = $$Generic;
```

Comment:
This is just a generic Svelte component type?

Reply:
Yup! See https://github.com/sveltejs/language-tools/issues/273#issuecomment-864576877